### PR TITLE
Add `Integer/toDouble`

### LINKED ISSUE #166: Double arithmetic?
I know it’s deliberately left out, but we have quite a few of these in config files:

```
, cookieMaxAge = 1209600.0 -- 14 * 24 * 60 * 60
```

As sometimes sub-second `NominalDiffTime`s are needed:

```haskell
instance Dhall.Interpret NominalDiffTime where
  autoWith _ = fromRational . toRational <$> Dhall.double
```

What would be the best way to go here?

### PR DESCRIPTION
**Pull request by Gabriel439**
Fixes #166

### CODE REVIEW

File: standard/semantics.md [hunk 1]
```diff
@@ -3619,13 +3645,17 @@ then that is a type error.
     Γ ⊢ ±n : Integer
 
 
-The built-in `Integer/show` function has the following type:
+The built-in functions on `Integer/show` have the following types:
```
Comment (michalrus) [inline standard/semantics.md]: Shouldn’t this be `Integer` instead of `Integer/show`?

Comment (Gabriel439) [reply standard/semantics.md]: Whoops!  Yes :)

Will fix

**Comment by f-f** [comment]
@Gabriel439 should we merge this? The implementation PR needs to be rebased, but this could go in already

**Comment by Gabriel439** [comment]
Yeah, I just forgot about this pull request, but it's merged now and I'll merge the matching Haskell change soon

**Pull request merged**